### PR TITLE
ui: watchlist bulk import + pagination

### PR DESCRIPTION
**Pull request by chikeichan**
relates to #178

### CODE REVIEW

File: app/pages/Watching/index.js [hunk 1]
```diff
@@ -54,31 +129,101 @@ class Watching extends Component {
...
+
+    if (isConfirmingReset) {
+      return 'This will permanently erased your watchlist on your device!';
+    }
+
+    if (isImporting) {
+      return 'Importing watchlist...';
+    }
+
+    return 'Your watchlist does not transfer across devices!';
```
Comment (sdtsui) [inline app/pages/Watching/index.js:160]: Can we leave this blank?  I feel like if someone is installing a desktop application and thinks if they install it again on another computer we magically know what they did...actually gets annoyed at us...that's ok.

File: app/pages/Watching/index.js [hunk 2]
```diff
@@ -54,31 +129,101 @@ class Watching extends Component {
...
   onClose = () => this.setState({isAddingName: false, name: ''});
 
+  getText = () => {
+    const {
+      isConfirmingReset,
+      isImporting,
+    } = this.state;
+
+    if (isConfirmingReset) {
+      return 'This will permanently erased your watchlist on your device!';
```
Comment (sdtsui) [inline app/pages/Watching/index.js:153]: plural 'erase**d**'

suggest:
'Unless previously exported, this will permanently delete your watchlist.'

**Review by sdtsui — COMMENTED** [review]
Something's up with the search.  Can you try to repro?  I recorded this after doing an import and an export. It might be some inconsistency between spreadsheet data not being fully saved in redux.  Note that 'omg' was not on the imported list but still showed up in search.

derp

**Comment by chikeichan** [comment]
Try it with [400 Most Common English Word](https://raw.githubusercontent.com/pkLazer/password_rank/master/4000-most-common-english-words-csv.csv)

search

**Pull request merged**